### ISSUE TITLE
Allow admins to unblock someone easily without having to go into the database

### ISSUE THREAD

**Issue by herbdool** [backdrop/backdrop-issues#3111]
Incorporate [Flood Unblock](https://www.drupal.org/project/flood_unblock) to allow admins to unblock someone easily without having to go into the database.

Related to #3110.

**Comment by laryn** [comment]
I agree this would be very useful. I actual just tracked down [your port of Flood Unblock](https://github.com/herbdool/flood_unblock), @herbdool -- is that ready for a release/usable in your opinion?

**Comment by herbdool** [comment]
@laryn from what I recall it is ready. I can move it to contrib. I'm not sure if I want to be a maintainer, though it's a pretty simple module.

**Comment by herbdool** [comment]
Done, I've moved it. @laryn do you want to take it for a spin before a release is made?

**Comment by laryn** [comment]
Sure, I'll give it a whirl.
